common/tier4_camera_view_rviz_plugin/src/bird_eye_view_controller.cpp

```diff
@@ -178,20 +179,12 @@ void BirdEyeViewController::update(float dt, float ros_dt)
   updateCamera();
 }
 
+// cppcheck-suppress unusedFunction
 void BirdEyeViewController::lookAt(const Ogre::Vector3 & point)
 {
   setPosition(point - target_scene_node_->getPosition());
 }
 
-void BirdEyeViewController::onTargetFrameChanged(
```

Comment:
This function is used in https://github.com/ros2/rviz/blob/1d031d285bd8e3294fc24b474b61b960269eb821/rviz_common/src/rviz_common/frame_position_tracking_view_controller.cpp#L110.

Please suppress this also

Reply:
@Owen-Liuyuxuan 

Thanks for the advice. I have changed it to suppression in the comments.